model/riscv_vmem_tlb.sail

```diff
@@ -1,124 +1,175 @@
-/*  Technology) under DARPA/AFRL contract FA8650-18-C-7809 ("CIFV"), and                 */
-/*  under DARPA contract HR0011-18-C-0016 ("ECATS") as part of the DARPA                 */
-/*  SSITH research programme.                                                            */
-/*                                                                                       */
-/*  This project has received funding from the European Research Council                 */
-/*  (ERC) under the European Union’s Horizon 2020 research and innovation                */
-/*  programme (grant agreement 789108, ELVER).                                           */
-/*                                                                                       */
-/*                                                                                       */
-/*  Redistribution and use in source and binary forms, with or without                   */
-/*  modification, are permitted provided that the following conditions                   */
-/*  are met:                                                                             */
-/*  1. Redistributions of source code must retain the above copyright                    */
-/*     notice, this list of conditions and the following disclaimer.                     */
-/*  2. Redistributions in binary form must reproduce the above copyright                 */
-/*     notice, this list of conditions and the following disclaimer in                   */
-/*     the documentation and/or other materials provided with the                        */
-/*     distribution.                                                                     */
-/*                                                                                       */
-/*  THIS SOFTWARE IS PROVIDED BY THE AUTHOR AND CONTRIBUTORS ``AS IS''                   */
-/*  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED                    */
-/*  TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A                      */
-/*  PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE AUTHOR OR                  */
-/*  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,                         */
-/*  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT                     */
-/*  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF                     */
-/*  USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND                  */
-/*  ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,                   */
-/*  OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT                   */
-/*  OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF                   */
-/*  SUCH DAMAGE.                                                                         */
+/*  RISCV Sail Model									 */
```

Comment:
I don’t think I understand the display above.  The above cited excerpt looks like this:

    /*  OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT			 */
    /*  OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF			 */
    /*  SUCH DAMAGE.									 */
    /*=======================================================================================*/

    // Although a TLB is not part of the RISC-V Architecture
    // specification, we model a simple TLB so that

The “RISCV Sail Model” line is on line 2, not after line 68.

The whitespace looks ok on my Emacs screen.

In VSCode, this whole header section initially looks messed up, because by default it has a “Spaces: 2” setting. On the bottom line, where it says “Spaces: 2”, this needs to be changed to “Indent using spaces” and setting the tab size to 8, to match the way Emacs displays it.

Reply:
I think some spaces have been replaced with hard tabs.

Reply:
PS: I highly recommend setting `Render Whitespace` to `boundary` in VSCode.

![image](https://github.com/riscv/sail-riscv/assets/376842/2332cdd3-25ce-422a-9c6c-0ffaeceea553)

Reply:
> I think some spaces have been replaced with hard tabs.

Is that a problem?  CODE_STYLE.md is silent on any policy regarding tabs vs. spaces.

Reply:
> PS: I highly recommend setting `Render Whitespace` to `boundary` in VSCode.

I rely on Emacs’ ’tabify’ and ‘untabify’ functions to do tabs/spaces properly for a selected region or buffer.

If Emacs and VSCode (and possibly vi) do it differently, I suggest not relying either one as a standard, as long as each tools is able to display it properly.